resource-watch/resource-watch
# Azavea development environment updates

## Overview
We needed to make a few small changes to the development environment to get things working on our end, primarily adjustments to how the Docker-based development setup worked. 

Additionally, this adds issue templates; this is primarily so that WRI has a template to follow when reporting issues during testing, but it can also be expanded to cover other uses later.

We've also merged our standard PR template into the existing PR template for this project. We tried to make intelligent choices about combining the two, but if something doesn't seem right, please let me know!

## Testing instructions
- I'm not sure whether the Docker development functionality is used at Vizzuality, but if you (or someone else) uses it, then it would be good to follow the Docker setup directions in the README and make sure that everything continues to work as expected.
- The other changes are just text, so reading through them to make sure they look okay should be enough.

## Jira task
N/A

## Notes
- I wasn't sure who to assign this to; if it's wrong please let me know or re-assign!
- @ethanroday you may want to take a look at the issue templates

---

## Checklist before submitting
